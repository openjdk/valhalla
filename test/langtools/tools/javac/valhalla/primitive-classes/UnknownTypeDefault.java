/*
 * @test /nodynamiccopyright/
 * @bug 8264216
 * @summary [lworld] unknown.Class.default gives misleading compilation error
 * @compile/fail/ref=UnknownTypeDefault.out -Xlint:all -Werror -XDrawDiagnostics -XDdev UnknownTypeDefault.java
 */

public class UnknownTypeDefault {

    public static void main(String [] args) {
        Object d1 = Y.default;
        Object d2 = y.Z.default;
    }
}
