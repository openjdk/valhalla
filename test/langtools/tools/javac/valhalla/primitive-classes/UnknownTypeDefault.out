UnknownTypeDefault.java:11:22: compiler.err.cant.resolve.location: kindname.class, Y, null, null, (compiler.misc.location: kindname.class, UnknownTypeDefault, null)
UnknownTypeDefault.java:12:24: compiler.err.cant.resolve.location: kindname.class, y.Z, null, null, (compiler.misc.location: kindname.class, UnknownTypeDefault, null)
2 errors
